client_it_test.go

```diff
@@ -208,42 +207,42 @@ func TestClientEventHandlingOrder(t *testing.T) {
 	// Create custom cluster, and client from it
 	cls := it.StartNewClusterWithOptions("event-order-test-cluster", 15701, it.MemberCount())
 	defer cls.Shutdown()
-	conf := cls.DefaultConfig()
+	cfg := cls.DefaultConfig()
 	ctx := context.Background()
-	c := it.MustValue(hz.StartNewClientWithConfig(ctx, conf)).(*hz.Client)
+	c := it.MustValue(hz.StartNewClientWithConfig(ctx, cfg)).(*hz.Client)
 	defer c.Shutdown(ctx)
-	ss := it.MustValue(serialization.NewService(&conf.Serialization)).(*serialization.Service)
 	// Create test map
-	m := it.MustValue(c.GetMap(ctx, "TestClientEventHandlingOrder")).(*hz.Map)
+	m := it.MustValue(c.GetMap(ctx, "my-map")).(*hz.Map)
```

Comment:
Could you use `NewUniqueObjectName` to generate the map name to make sure there's no conflict with other test runs? Something like:
```
ls := "smart"
if !smart {
	ls = "unisocket"
}
mapName := NewUniqueObjectName("map", ls)
```

Reply:
Thanks for pointing that out! Used "it.MapTester" utility for the test at https://github.com/hazelcast/hazelcast-go-client/commit/a7fc2045715f84ad7e4a13b2dbc74d77857b3590